curs2java/src/persons/Person.java

```diff
@@ -0,0 +1,37 @@
+package persons;
+
+public class Person {
+	//Persons have a first name a last name and an age. 
+	private String FirstName;
+	private String LastName;
+	private int varsta;
+	public String getFirstName() {
+		return FirstName;
+	}
+	public void setFirstName(String firstName) {
+		FirstName = firstName;
+	}
+	public String getLastName() {
+		return LastName;
+	}
+	public void setLastName(String lastName) {
+		LastName = lastName;
+	}
+	public int getVarsta() {
+		return varsta;
+	}
+	public void setVarsta(int varsta) {
+		this.varsta = varsta;
+	}
+	public Person(String firstName, String lastName, int varsta) {
```

Comment:
By convention, the constructor usually is the first in front of the rest of the methods in the class.